ScoutSuite/frontend/src/pages/Dashboard/TabsContent/Summary/index.js

```diff
@@ -11,7 +11,7 @@ const propTypes = {
 };
 
 const Summary = props => {
-  const services = props.services.sort((a, b) => b.flagged_items - a.flagged_items);
+  const services = props.services.sort((a, b) => b.issues.High - a.issues.High);
```

Comment:
we should also use `lodash/sortBy` here so we can sort for each security level.

Reply:
Fixed :)

![image](https://user-images.githubusercontent.com/1829117/110400373-85c04080-8045-11eb-9140-749c15e4b8bd.png)